src/makeRobot.js

```diff
@@ -27,18 +27,71 @@
  *
  * @typedef {object} Robot
  * @property {string} name
- * @property {number} wheels
- * @property {number} version
+ * @property {stepber} wheels
+ * @property {stepber} version
  * @property {function} info
  *
  * @param {string} name
- * @param {number} wheels
- * @param {number} version
+ * @param {stepber} wheels
+ * @param {stepber} version
  *
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    info: `name: ${name}, chip version: ${version}, wheels: ${wheels}`,
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(step = 1) {
+      if (step < 0) {
+        return robot;
+      }
+      this.coords.y += step;
+
+      return robot;
```

Comment:
Таким чином, ти уникнеш повторення
```suggestion
      if (step > 0) {
        this.coords.y += step;
      }

      return robot;
```